client/state/sites/selectors/get-jetpack-site-update-files-disabled-reasons.js

```diff
@@ -31,8 +30,8 @@ export default function getJetpackSiteUpdateFilesDisabledReasons(
 
 	const fileModDisabled = getSiteOption( state, siteId, 'file_mod_disabled' );
 
-	return compact(
-		fileModDisabled.map( ( clue ) => {
+	return fileModDisabled
```

Comment:
It's weird that we don't check if `fileModDisabled` exists before calling `.map()` on it. We seem to be checking it in other instances in this PR. Should we do it here as well?

Reply:
That's true, and the reason why it never crashed for anyone is that the selector was never used since it was added 5 years ago. Instead, we're using the similar `getSiteFileModDisableReason( site )` function. I'll remove the selector.